### PR TITLE
Fix 5.0 beta30 docs

### PR DESCRIPTION
**Pull request by rodesai**
### Description 
_What behavior do you want to change, why, how does your patch achieve the changes?_

### Testing done 
_Describe the testing stategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: docs/includes/ksql-includes.rst [hunk 1]
```diff
@@ -293,12 +293,12 @@ the latest offset.
...
-        ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+        Query ID                        | Kafka Topic              | Query String
+        ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+        CTAS_PAGEVIEWS_REGIONS_3        | PAGEVIEWS_REGIONS        | CREATE TABLE pageviews_regions WITH (value_format='avro') AS SELECT gender, regionid , COUNT(*) AS numusers FROM pageviews_female WINDOW TUMBLING (size 30 second) GROUP BY gender, regionid HAVING COUNT(*) > 1;
```
Comment (apurvam) [inline docs/includes/ksql-includes.rst]: Why has this changed?

Comment (hjafarpour) [reply docs/includes/ksql-includes.rst]: These are because of the new query id assignment that adds a counter to make sure query id is unique.

Comment (apurvam) [reply docs/includes/ksql-includes.rst]: 👍

Comment (miguno) [reply docs/includes/ksql-includes.rst]: Thanks for paying attention to this, @hjafarpour !

**Pull request merged**